modules/options/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -176,6 +177,24 @@ object Artifacts {
           ).left.flatMap(_.maybeRecoverWithDefault(Seq.empty, maybeRecoverOnError))
         }
 
+        val bridgeJarsOpt = Option.when(
+          scalaArtifactsParams.params.scalaVersion.startsWith("3.") && includeBuildServerDeps
```

Comment:
Would we not want to do it always?

Reply:
We don't need the bridge when Bloop is not used, or when working with Scala older than 3.x.y